apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/creating-a-site.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Creating a Site | Sitecore Accelerate for partners'
+description: 'How to create your Site and Site Collection in XM Cloud'
+area: ['accelerate']
+hasSubPageNav: true
+hasInPageNav: false
+---
+
+## Problem
+
+I need to create a new Site, what is the best way to do it? Should it be in a Site Collection? Should I create a new site collection or put it in an existing one?
+
+## Solution
+
+When creating a Site in XM Cloud, a site collection is always required. Then Sites live in a site collection. You can have as many site collections as needed and a site collection can have many Sites.
+
+### Creating a Site Collection
+
+Before we create a site collection, we need to create an SXA Module for our templates and rendering items to live in. It is important to create the Headless Module before you create the site collection, once the site collection is created, it is not possible to create a `Headless Module`.
```

Comment:
If we are going to capitalize Site and Site Collection then I think we should be consistent. Or all lowercase, but again be consistent

Reply:
Fixex